docker/docker-compose.objects-apis.yml

```diff
@@ -20,28 +20,53 @@ services:
       - open-forms-dev
 
   objecttypes-web:
-    image: maykinmedia/objecttypes-api:${OBJECTTYPES_VERSION:-2.1.1}
+    image: maykinmedia/objecttypes-api:${OBJECTTYPES_VERSION:-2.2.0}
     environment:
       - DJANGO_SETTINGS_MODULE=objecttypes.conf.docker
       - SECRET_KEY=${SECRET_KEY:-fgv=c0hz&tl*8*3m3893@m+1pstrvidc9e^5@fpspmg%cy$15d}
       - ALLOWED_HOSTS=*
+      - USE_X_FORWARDED_HOST=true
       - DB_HOST=objects-objecttypes-db
+      - DB_NAME=objecttypes
+      - DB_USER=objecttypes
+      - CACHE_DEFAULT=objects-redis:6379/1
+      - CACHE_AXES=objects-redis:6379/1
+      - OBJECTTYPE_SUPERUSER_USERNAME=admin
+      - OBJECTTYPE_SUPERUSER_PASSWORD=admin
+      - OBJECTTYPE_SUPERUSER_EMAIL=admin@localhost
+      - DISABLE_2FA=true
+    volumes:
+      # mount fixtures dir to automatically populate the DB
+      - ./objects-apis/fixtures/objecttypes_api_fixtures.json:/app/fixtures/objecttypes_api_fixtures.json
     depends_on:
       - objects-objecttypes-db
+      - objects-redis
     networks:
       - open-forms-dev
 
   objects-web:
-    image: maykinmedia/objects-api:${OBJECTS_VERSION:-2.3.0}
+    image: maykinmedia/objects-api:${OBJECTS_VERSION:-2.3.2}
```

Comment:
I don't see the `v2.3.2` in the repo. Can we assume that from somewhere else?

Reply:
It is available here: https://hub.docker.com/r/maykinmedia/objects-api/tags

Reply:
yeah I had to look at docker hub / git tags too, it's not in the changelog